### PR TITLE
Support of Symfony flex sites

### PR DESCRIPTION
**Pull request by venimus**
The new symfony structure (Flex) recommends index.php as front controller and public in `/public` instead of `/web`.

+ Allow configuration of `project_path`
+ Use `nginx_site_default_root`
+ allow installing of multiple php versions with limitations. Will allow reprovision with changed php_fpm_version. The latest will be used by nginx, however the previous services will stay active.

### CODE REVIEW

**Comment by venimus** [comment]
@dincho This is my idea. Not sure if this is the optimal solution to handle both

File: ansible/services.yml [hunk 1]
```diff
@@ -76,11 +76,16 @@
          - listen 80 default_server
          - listen 443 ssl http2
          - server_name _
-         - root /app/web
+         - |
+          set $root /app/web;
```
Comment (dincho) [inline ansible/services.yml]: What if we have this as extra_var? Default should be the new path?

**Review by dincho — COMMENTED** [review]
LGTM overall

File: ansible/services.yml (hunk 1 above)
Comment (venimus) [reply ansible/services.yml]: this var is nginx. do you mean to try to add/use ansible var?

Comment (dincho) [reply ansible/services.yml]: Yeah, ansible var used in this config

**Comment by venimus** [comment]
@dincho improved it a lot. Tested it with different setups (including 5.6)

**Pull request merged**